packages/client/hmi-client/src/components/workflow/ops/model/tera-model-node.vue

```diff
@@ -93,6 +94,16 @@ onMounted(async () => {
 		await getModelById(state.modelId);
 	}
 });
+
+watch(
+	() => props.node.active,
+	async () => {
+		const newModelId = props.node.outputs.find((d) => d.id === props.node.active)?.value?.[0];
+		if (newModelId) {
```

Comment:
Should check that the `newModelId` is not equal to the current `model.value.id`.